installation/scripts/testing-common.sh

```diff
@@ -7,43 +7,68 @@ function context_arg() {
     fi
 }
 
-function printLogsFromFailedHelmTests() {
-    local namespace=$1
+function cmdGetPodsForSuite() {
+    local suiteName=$1
+    cmd="kubectl $(context_arg) get pods -l testing.kyma-project.io/suite-name=${suiteName} \
+            --all-namespaces \
+            --no-headers=true \
+            -o=custom-columns=name:metadata.name,ns:metadata.namespace \
+            --show-all=true"
+    echo $cmd
+}
+
+function printLogsFromFailedTests() {
```

Comment:
IMO we can improve our test output:

```
Test suite 'testsuite-all-2019-04-08-15-27' succeeded.
ClusterTestSuite details:
apiVersion: testing.kyma-project.io/v1alpha1
kind: ClusterTestSuite
metadata:
  annotations:
    kubectl.kubernetes.io/last-applied-configuration: |
      {"apiVersion":"testing.kyma-project.io/v1alpha1","kind":"ClusterTestSuite","metadata":{"annotations":{},"labels":{"controller-tools.k8s.io":"1.0"},"name":"testsuite-all-2019-04-08-15-27","namespace":""},"spec":{"concurrency":1,"count":1,"maxRetries":0}}
  creationTimestamp: 2019-04-08T15:27:10Z
  generation: 1
  labels:
    controller-tools.k8s.io: "1.0"
  name: testsuite-all-2019-04-08-15-27
  resourceVersion: "15125"
  selfLink: /apis/testing.kyma-project.io/v1alpha1/clustertestsuites/testsuite-all-2019-04-08-15-27
  uid: c6ca340d-5a12-11e9-be1f-42010a9c007a
spec:
  concurrency: 1
  count: 1
  maxRetries: 0
status:
  completionTime: 2019-04-08T15:52:36Z
  conditions:
  - status: "False"
    type: Running
  - status: "True"
    type: Succeeded
  results:
  - executions:
    - completionTime: 2019-04-08T15:27:16Z
      id: oct-tp-testsuite-all-2019-04-08-15-27-test-dex-dex-connection-dex-0
      podPhase: Succeeded
      startTime: 2019-04-08T15:27:10Z
    name: test-dex-dex-connection-dex
    namespace: kyma-system
    status: Succeeded
  - executions:
    - completionTime: 2019-04-08T15:28:03Z
      id: oct-tp-testsuite-all-2019-04-08-15-27-test-api-controller-acceptance-0
      podPhase: Succeeded
      startTime: 2019-04-08T15:27:16Z
    name: test-api-controller-acceptance
    namespace: kyma-system
    status: Succeeded
  - executions:
    - completionTime: 2019-04-08T15:30:31Z
      id: oct-tp-testsuite-all-2019-04-08-15-27-test-core-core-acceptance-0
      podPhase: Succeeded
      startTime: 2019-04-08T15:28:04Z
    name: test-core-core-acceptance
    namespace: kyma-system
    status: Succeeded
  - executions:
    - completionTime: 2019-04-08T15:30:43Z
      id: oct-tp-testsuite-all-2019-04-08-15-27-test-core-namespace-controller-0
      podPhase: Succeeded
      startTime: 2019-04-08T15:30:32Z
    name: test-core-namespace-controller
    namespace: kyma-system
    status: Succeeded
  - executions:
    - completionTime: 2019-04-08T15:31:29Z
      id: oct-tp-testsuite-all-2019-04-08-15-27-test-event-bus-tester-0
      podPhase: Succeeded
      startTime: 2019-04-08T15:30:43Z
    name: test-event-bus-tester
    namespace: kyma-system
    status: Succeeded
  - executions:
    - completionTime: 2019-04-08T15:36:29Z
      id: oct-tp-testsuite-all-2019-04-08-15-27-application-registry-tests-0
      podPhase: Succeeded
      startTime: 2019-04-08T15:31:30Z
    name: application-registry-tests
    namespace: kyma-integration
    status: Succeeded
  - executions:
    - completionTime: 2019-04-08T15:38:32Z
      id: oct-tp-testsuite-all-2019-04-08-15-27-test-core-kubeless-0
      podPhase: Succeeded
      startTime: 2019-04-08T15:36:30Z
    name: test-core-kubeless
    namespace: kyma-system
    status: Succeeded
  - executions:
    - completionTime: 2019-04-08T15:38:38Z
      id: oct-tp-testsuite-all-2019-04-08-15-27-test-core-apiserver-proxy-0
      podPhase: Succeeded
      startTime: 2019-04-08T15:38:32Z
    name: test-core-apiserver-proxy
    namespace: kyma-system
    status: Succeeded
  - executions:
    - completionTime: 2019-04-08T15:45:34Z
      id: oct-tp-testsuite-all-2019-04-08-15-27-test-core-core-ui-acceptance-0
      podPhase: Succeeded
      startTime: 2019-04-08T15:38:39Z
    name: test-core-core-ui-acceptance
    namespace: kyma-system
    status: Succeeded
  - executions:
    - completionTime: 2019-04-08T15:45:51Z
      id: oct-tp-testsuite-all-2019-04-08-15-27-connection-token-handler-tests-0
      podPhase: Succeeded
      startTime: 2019-04-08T15:45:34Z
    name: connection-token-handler-tests
    namespace: kyma-integration
    status: Succeeded
  - executions:
    - completionTime: 2019-04-08T15:49:29Z
      id: oct-tp-testsuite-all-2019-04-08-15-27-test-core-kubeless-int-0
      podPhase: Succeeded
      startTime: 2019-04-08T15:45:51Z
    name: test-core-kubeless-int
    namespace: kyma-system
    status: Succeeded
  - executions:
    - completionTime: 2019-04-08T15:49:51Z
      id: oct-tp-testsuite-all-2019-04-08-15-27-test-monitoring-0
      podPhase: Succeeded
      startTime: 2019-04-08T15:49:29Z
    name: test-monitoring
    namespace: kyma-system
    status: Succeeded
  - executions:
    - completionTime: 2019-04-08T15:50:07Z
      id: oct-tp-testsuite-all-2019-04-08-15-27-test-assetstore-assetstore-integration-0
      podPhase: Succeeded
      startTime: 2019-04-08T15:49:51Z
    name: test-assetstore-assetstore-integration
    namespace: kyma-system
    status: Succeeded
  - executions:
    - completionTime: 2019-04-08T15:52:36Z
      id: oct-tp-testsuite-all-2019-04-08-15-27-application-operator-tests-0
      podPhase: Succeeded
      startTime: 2019-04-08T15:50:07Z
    name: application-operator-tests
    namespace: kyma-integration
    status: Succeeded
  startTime: 2019-04-08T15:27:10Z
Test summary
Test status: test-dex-dex-connection-dex - Succeeded
Test status: test-api-controller-acceptance - Succeeded
Test status: test-core-core-acceptance - Succeeded
Test status: test-core-namespace-controller - Succeeded
Test status: test-event-bus-tester - Succeeded
Test status: application-registry-tests - Succeeded
Test status: test-core-kubeless - Succeeded
Test status: test-core-apiserver-proxy - Succeeded
Test status: test-core-core-ui-acceptance - Succeeded
Test status: connection-token-handler-tests - Succeeded
Test status: test-core-kubeless-int - Succeeded
Test status: test-monitoring - Succeeded
Test status: test-assetstore-assetstore-integration - Succeeded
Test status: application-operator-tests - Succeeded
All test pods should be terminated. Checking...
Flag --show-all has been deprecated, will be removed in an upcoming release
OK
Flag --show-all has been deprecated, will be removed in an upcoming release
Testing 'oct-tp-testsuite-all-2019-04-08-15-27-application-operator-tests-0' from namespace 'kyma-integration'
Test of 'oct-tp-testsuite-all-2019-04-08-15-27-application-operator-tests-0' was successful
Logs are not displayed after success
End of testing 'oct-tp-testsuite-all-2019-04-08-15-27-application-operator-tests-0'
Testing 'oct-tp-testsuite-all-2019-04-08-15-27-application-registry-tests-0' from namespace 'kyma-integration'
Test of 'oct-tp-testsuite-all-2019-04-08-15-27-application-registry-tests-0' was successful
Logs are not displayed after success
End of testing 'oct-tp-testsuite-all-2019-04-08-15-27-application-registry-tests-0'
Testing 'oct-tp-testsuite-all-2019-04-08-15-27-connection-token-handler-tests-0' from namespace 'kyma-integration'
Test of 'oct-tp-testsuite-all-2019-04-08-15-27-connection-token-handler-tests-0' was successful
Logs are not displayed after success
End of testing 'oct-tp-testsuite-all-2019-04-08-15-27-connection-token-handler-tests-0'
Testing 'oct-tp-testsuite-all-2019-04-08-15-27-test-api-controller-acceptance-0' from namespace 'kyma-system'
Test of 'oct-tp-testsuite-all-2019-04-08-15-27-test-api-controller-acceptance-0' was successful
Logs are not displayed after success
End of testing 'oct-tp-testsuite-all-2019-04-08-15-27-test-api-controller-acceptance-0'
Testing 'oct-tp-testsuite-all-2019-04-08-15-27-test-assetstore-assetstore-integration-0' from namespace 'kyma-system'
Test of 'oct-tp-testsuite-all-2019-04-08-15-27-test-assetstore-assetstore-integration-0' was successful
Logs are not displayed after success
End of testing 'oct-tp-testsuite-all-2019-04-08-15-27-test-assetstore-assetstore-integration-0'
Testing 'oct-tp-testsuite-all-2019-04-08-15-27-test-core-apiserver-proxy-0' from namespace 'kyma-system'
Test of 'oct-tp-testsuite-all-2019-04-08-15-27-test-core-apiserver-proxy-0' was successful
Logs are not displayed after success
End of testing 'oct-tp-testsuite-all-2019-04-08-15-27-test-core-apiserver-proxy-0'
Testing 'oct-tp-testsuite-all-2019-04-08-15-27-test-core-core-acceptance-0' from namespace 'kyma-system'
Test of 'oct-tp-testsuite-all-2019-04-08-15-27-test-core-core-acceptance-0' was successful
Logs are not displayed after success
End of testing 'oct-tp-testsuite-all-2019-04-08-15-27-test-core-core-acceptance-0'
Testing 'oct-tp-testsuite-all-2019-04-08-15-27-test-core-core-ui-acceptance-0' from namespace 'kyma-system'
Test of 'oct-tp-testsuite-all-2019-04-08-15-27-test-core-core-ui-acceptance-0' was successful
Logs are not displayed after success
End of testing 'oct-tp-testsuite-all-2019-04-08-15-27-test-core-core-ui-acceptance-0'
Testing 'oct-tp-testsuite-all-2019-04-08-15-27-test-core-kubeless-0' from namespace 'kyma-system'
Test of 'oct-tp-testsuite-all-2019-04-08-15-27-test-core-kubeless-0' was successful
Logs are not displayed after success
End of testing 'oct-tp-testsuite-all-2019-04-08-15-27-test-core-kubeless-0'
Testing 'oct-tp-testsuite-all-2019-04-08-15-27-test-core-kubeless-int-0' from namespace 'kyma-system'
Test of 'oct-tp-testsuite-all-2019-04-08-15-27-test-core-kubeless-int-0' was successful
Logs are not displayed after success
End of testing 'oct-tp-testsuite-all-2019-04-08-15-27-test-core-kubeless-int-0'
Testing 'oct-tp-testsuite-all-2019-04-08-15-27-test-core-namespace-controller-0' from namespace 'kyma-system'
Test of 'oct-tp-testsuite-all-2019-04-08-15-27-test-core-namespace-controller-0' was successful
Logs are not displayed after success
End of testing 'oct-tp-testsuite-all-2019-04-08-15-27-test-core-namespace-controller-0'
Testing 'oct-tp-testsuite-all-2019-04-08-15-27-test-dex-dex-connection-dex-0' from namespace 'kyma-system'
Test of 'oct-tp-testsuite-all-2019-04-08-15-27-test-dex-dex-connection-dex-0' was successful
Logs are not displayed after success
End of testing 'oct-tp-testsuite-all-2019-04-08-15-27-test-dex-dex-connection-dex-0'
Testing 'oct-tp-testsuite-all-2019-04-08-15-27-test-event-bus-tester-0' from namespace 'kyma-system'
Test of 'oct-tp-testsuite-all-2019-04-08-15-27-test-event-bus-tester-0' was successful
Logs are not displayed after success
End of testing 'oct-tp-testsuite-all-2019-04-08-15-27-test-event-bus-tester-0'
Testing 'oct-tp-testsuite-all-2019-04-08-15-27-test-monitoring-0' from namespace 'kyma-system'
Test of 'oct-tp-testsuite-all-2019-04-08-15-27-test-monitoring-0' was successful
Logs are not displayed after success
End of testing 'oct-tp-testsuite-all-2019-04-08-15-27-test-monitoring-0'
clustertestsuite.testing.kyma-project.io "testsuite-all-2019-04-08-15-27" deleted
Images with tag latest are not allowed. Checking...
OK
```

as you can see we are printing 3 times that all tests passed - test suite, test summary, logs printing. IMO we can remote printing all those entries like: `Logs are not displayed after success` etc. Thanks to that output will be more compact :)